src/shared/logger/activation.ts

```diff
@@ -109,7 +110,7 @@ export function makeLogger(
         logPaths?: string[]
         outputChannels?: vscode.OutputChannel[]
         useDebugConsole?: boolean
-        useConsoleLog?: boolean
+        useDeveloperToolsConsole?: boolean
```

Comment:
The old name was probably better. "console" is a generally known concept in the JS ecosystem and in nodejs "developer tools" is not involved (not mentioned on https://nodejs.org/api/console.html for example).